kit/popups/src/ActionSheet/UIActionSheetAction.tsx

```diff
@@ -1,49 +1,52 @@
 import * as React from 'react';
-import { View } from 'react-native';
-import { UIBoxButton, UIBoxButtonType, UIBoxButtonVariant } from '@tonlabs/uikit.controls';
+import { StyleSheet } from 'react-native';
+import { UILabel, TypographyVariants, ColorVariants } from '@tonlabs/uikit.themes';
+import { UILayoutConstant } from '@tonlabs/uikit.layout';
 import { UIActionSheetActionProps, UIActionSheetActionType } from './types';
-
-const getActionVariant = (type: UIActionSheetActionType): UIBoxButtonVariant => {
-    switch (type) {
-        case UIActionSheetActionType.Negative:
-            return UIBoxButtonVariant.Negative;
-        case UIActionSheetActionType.Cancel:
-        case UIActionSheetActionType.Neutral:
-        case UIActionSheetActionType.Disabled:
-        default:
-            return UIBoxButtonVariant.Neutral;
-    }
-};
-
-const getBoxButtonType = (type: UIActionSheetActionType): UIBoxButtonType => {
-    switch (type) {
-        case UIActionSheetActionType.Cancel:
-            return UIBoxButtonType.Nulled;
-        case UIActionSheetActionType.Negative:
-        case UIActionSheetActionType.Neutral:
-        case UIActionSheetActionType.Disabled:
-        default:
-            return UIBoxButtonType.Tertiary;
-    }
-};
+import { UIForeground } from '../UIForeground';
+import { TouchableWrapper } from '../TouchableWrapper';
 
 export const UIActionSheetAction: React.FC<UIActionSheetActionProps> = ({
     type,
     title,
     onPress,
 }: UIActionSheetActionProps) => {
-    const variant: UIBoxButtonVariant = getActionVariant(type);
-    const boxButtonType: UIBoxButtonType = getBoxButtonType(type);
-    return (
-        <View key={title}>
-            <UIBoxButton
+    if (type === UIActionSheetActionType.Cancel) {
+        return (
+            <TouchableWrapper
```

Comment:
Why it isn't done with Foreground?

Reply:
It is done

Reply:
Because this button differs from the UIForeground in font and color

Reply:
Done